DataFormats/interface/MET.h

```diff
@@ -31,6 +31,14 @@ namespace cat {
     float unclusteredEnSumEt(int dir) const { if (dir > 0) return unclusteredEnUp_sumet_; else return unclusteredEnDown_sumet_;}
     float unclusteredEnPt(int dir) const { return hypotf(unclusteredEnPx(dir), unclusteredEnPy(dir));}
     float unclusteredEnPhi(int dir) const { return std::atan2(unclusteredEnPy(dir), unclusteredEnPx(dir));}
+
+    float JetEnPx(int dir) const { if (dir > 0) return jetEnUp_px_; else return jetEnDown_px_;}
+    float JetEnPy(int dir) const { if (dir > 0) return jetEnUp_py_; else return jetEnDown_py_;}
+    float JetEnSumEt(int dir) const { if (dir > 0) return jetEnUp_sumet_; else return jetEnDown_sumet_;}
+    float JetResPx(int dir) const { if (dir > 0) return jetResUp_px_; else return jetResDown_px_;}
+    float JetResPy(int dir) const { if (dir > 0) return jetResUp_py_; else return jetResDown_py_;}
+    float JetResSumEt(int dir) const { if (dir > 0) return jetResUp_sumet_; else return jetResDown_sumet_;}
```

Comment:
I assumed since this was to get a  systematic change in MET no one would use JetEnPx(0). I changed it so that you have to enter 1. or -1. to get the up/down systematic shifts. Anything else will return 0.